### ISSUE TITLE
 SELECT2_THEME setting is required in 7.9.0, witch breaks existing projects

### ISSUE THREAD

**Issue by wikti** [codingjoe/django-select2#113]
**Describe the bug**
In new version, new setting ``SELECT2_THEME`` is required and raises error when not provided. This setting should have default value: ``default`` for backward compatibility.

**Exception & Traceback**
Revalent part of traceback:
```
/usr/local/lib/python3.8/site-packages/django_select2/forms.py:261: in build_attrs
    attrs = super().build_attrs(default_attrs, extra_attrs=extra_attrs)
/usr/local/lib/python3.8/site-packages/django_select2/forms.py:93: in build_attrs
    "data-theme": settings.SELECT2_THEME,
/usr/local/lib/python3.8/site-packages/django/conf/__init__.py:83: in __getattr__
    val = getattr(self._wrapped, name)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <UserSettingsHolder>, name = 'SELECT2_THEME'
    def __getattr__(self, name):
        if not name.isupper() or name in self._deleted:
            raise AttributeError
>       return getattr(self.default_settings, name)
E       AttributeError: 'Settings' object has no attribute 'SELECT2_THEME'
/usr/local/lib/python3.8/site-packages/django/conf/__init__.py:239: AttributeError
```

**To Reproduce**
In test remove attribute ``SELECT2_THEME`` from settings. This mimics the behavior of project updating this library.

**Comment by codingjoe** [comment]
cc @CleitonDeLima

**Comment by CleitonDeLima** [comment]
hi @wikti 

I use the most up-to-date version of `django-select2` (7.9.0) on projects that don't specify `SELECT2_THEME`, do you use a third party to manage your settings?

**Comment by wikti** [comment]
Sorry for misleading issue.

There is no problem when application running normally, but tests failed with exception provided above. After poking around, I found why I got this message. The problem is between `pytest-django` settings fixture and `django-appconf` configuration. The problem was described in https://github.com/django-compressor/django-appconf/issues/30#issuecomment-195374791. After making suggested fix, tests have passed.
